ev3dev2/__init__.py

```diff
@@ -28,6 +28,15 @@
 if sys.version_info < (3,4):
     raise SystemError('Must be using Python 3.4 or higher')
 
+def is_micropython():
+    return sys.implementation.name == "micropython"
+
+def chain_exception(exception, cause):
```

Comment:
I'm not sure if I like how I did this... Micropython doesn't support exception chaining, so we have to suppress the `from` when running on that interpreter. But now `chain_exception` shows up in the call stack.

Reply:
I guess we could implement our own Exception subclass, that would carry its parent exception around, but I am not sure if that would be any better than what you did.